static/downloadables/calculate_score.py

```diff
@@ -276,20 +277,25 @@ def formatJson(studyInfo, outputFile):
 
 
 def createMarks(betas, nonMissingSnps, studyID, mark, valueType):
-    prs = str(getPRSFromArray(betas, nonMissingSnps, valueType))
+    prs = str(getPRSFromArray(betas, nonMissingSnps, valueType, studyID))
     # Add a mark to studies that have duplicate snps with varying pvalue annotations
     if mark is True:
         studyID = studyID + '†'
     return prs, studyID
 
 
-def getPRSFromArray(betas, nonMissingSnps, valueType):
+def getPRSFromArray(betas, nonMissingSnps, valueType, studyID):
+    valueType = valueType.lower()
 # calculate the PRS from the list of betas
     ploidy = 2
     combinedBetas = 0
     # if there are no values in the betas array, set combinedBetas to 'NF'
     if not betas:
         combinedBetas = "NF"
+    elif nonMissingSnps == 0:
+        combinedBetas = "NF"
+        SystemExit(f"ERROR: We had values that were zero, so the snps weren't added to the lists. Please let us know about this error and send up this studyID: {studyID}")
```

Comment:
This error is a little confusing for me...so i imagine a user will be really confused. What values were zero? What lists were the snps not added to?